src/dracula.yml

```diff
@@ -5,29 +5,35 @@ maintainers:
 semanticClass: theme.dracula
 dracula:
   base:
-    - &BG '#282A36'
-    - &FG '#F8F8F2'
+    - &BG        '#282A36'
+    - &FG        '#F8F8F2'
     - &SELECTION '#44475A'
+    - &CYAN      '#8BE9FD'
+    - &GREEN     '#50FA7B'
+    - &ORANGE    '#FFB86C'
+    - &PINK      '#FF79C6'
+    - &PURPLE    '#BD93F9'
+    - &RED       '#FF5555'
+    - &YELLOW    '#F1FA8C'
   ansi:
-    - &BLACK   '#4D4D4D'
-    - &RED     '#FF5555'
-    - &GREEN   '#50FA7B'
-    - &YELLOW  '#F1FA8C'
-    - &PURPLE  '#BD93F9'
-    - &PINK    '#FF79C6'
-    - &CYAN    '#8BE9FD'
-    - &COLOR7  '#BFBFBF'
-    - &COLOR8  '#575757'
-    - &COLOR9  '#FF6E67'
-    - &COLOR10 '#5AF78E'
-    - &COLOR11 '#F4F99D'
-    - &COLOR12 '#CAA9FA'
-    - &COLOR13 '#FF92D0'
-    - &COLOR14 '#9AEDFE'
+    - &COLOR0  '#44475A'
+    - &COLOR1  '#DE9442'
```

Comment:
Can we adjust this color some more (along with its `COLOR9` counterpart)? Currently it appears orange. It should be clearly red.

Reply:
Oh, whoops! Looks like I copied the orange hex values instead of the red ones. I'll update that.